integration/client/apps/update_test.go

```diff
@@ -30,7 +30,7 @@ func TestUpdatePull(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	app, err := c.AppRun(ctx, "foo", &client.AppRunOptions{Name: "test"})
+	app, err := c.AppRun(ctx, "foo:latest", &client.AppRunOptions{Name: "test"})
```

Comment:
What is the necessity of adding `latest` here?

Reply:
Doesn't seem to pass without it. Will need to look into this more.

Reply:
Fixed.